docs/roadmap/index.html

```diff
@@ -69,28 +63,29 @@ <h2 id="next-steps">Next Steps</h2>
   <li>propose a new charter for a W3C WebAssembly Working Group</li>
   <li>graduate the <a href="https://github.com/llvm-mirror/llvm/tree/master/test/CodeGen/WebAssembly">WebAssembly LLVM backend</a> from experimental to stable (and update Emscripten)</li>
   <li>prototype additional WebAssembly integration into browser developer tools</li>
-  <li>finalize a WebAssembly <a href="https://github.com/WebAssembly/design/issues/112">logo</a></li>
   <li>Start work on <a href="/docs/future-features/">post-MVP features</a></li>
-  <li>Create a publicly-runnable conformance test suite</li>
 </ul>
 
-<h2 id="planned-changes-before-release">Planned Changes Before Release</h2>
+<h2 id="changes-since-the-browser-preview">Changes Since the Browser Preview</h2>
 
 <ul>
-  <li>At the end of the Browser Preview period, the
-	<a href="/docs/binary-encoding/#high-level-structure">binary version</a>
-	will be reset to <code class="highlighter-rouge">0x1</code> (at which point the version will be frozen and all new features will be added in a backwards-compatible manner, and be detected through feature-testing).</li>
+  <li>The <a href="/docs/binary-encoding/#high-level-structure">binary version</a>
+  of WebAssembly has been frozen at <code class="highlighter-rouge">0x1</code> (from this point forward
+  all new features will be added in a backwards-compatible manner,
+  and be detected through feature-testing). To update your
+  WebAssembly modules, upgrade to the latest version of Emscripten 
+  and recompile.</li>
```

Comment:
I added a link to the advanced tools page in the message above. Happy to add more tools to that page as stuff comes online (like the gcc backend).

I think the roadmap page isn't the best place for the transition and #1006 stuff... not sure where would be better. Maybe a FAQ under getting started?

I think 0x1 is in 1.37.3, but I'll hold off mentioning a specific version until the async-by-default changes get tagged.